ocgis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+    name: ocgis
+    version: 1.2.0
+
+source:
+    git_url: https://github.com/NCPP/ocgis.git
+    git_tag: v1.2.0
+
+build:
+    number: 0
+    skip: True # [win]
+    skip: True # [osx]
+    skip: True # [py3k]
+
+requirements:
+    build:
+        - python >=2.7,<3
+        - cfunits-python
+        - fiona
+        - gdal <2
+        - libgdal <2
+        - netcdf4 <=1.1.6
```

Comment:
Got it. Release should be next month. An IOOS-hosted ``gdal`` and ``fiona`` will be awesome!